docs/source/index.rst

```diff
@@ -1,81 +1,51 @@
+CellRank's key applications
+---------------------------
+- compute initial, terminal and intermediate :doc:`macrostates <notebooks/tutorials/initial_terminal_states>`
+  :cite:`reuter:18,reuter:22`.
+- infer :doc:`fate probabilities <notebooks/tutorials/fate_probabilities>` towards terminal states.
+- visualize :doc:`gene expression trends <notebooks/tutorials/gene_trends>` along specific trajectories.
+- identify potential :doc:`driver genes <notebooks/tutorials/fate_probabilities>` for each trajectory.
+- ... and much more, check out our :doc:`API <api>`.
 
 Getting started with CellRank
 -----------------------------
-If you're new to CellRank, make sure to go though the `basic tutorial`_ which introduces you to CellRank's high-level
-API. Most biological systems require a bit more control, so be sure to check out the `kernels and estimators tutorial`_
-which allows to unlock the full power of CellRank. If you want to see individual functions in action, visit our
-`gallery`_.
+We have :doc:`tutorials` and :doc:`examples` that help you getting started. Tutorials are longer and explain
+computational pipelines, examples are short and demonstrate individual steps. To learn more about the principles
+behind CellRank, visit :doc:`about CellRank <about/index>`.
 
-To use CellRank without RNA velocity information, check out the `beyond RNA velocity tutorial`_ as well as the
-`time-series tutorial`_.
-
-CellRank's key applications
----------------------------
-- compute initial & terminal as well as intermediate `macrostates`_ of your biological system
-- infer `fate probabilities`_ towards the terminal states for each individual cell
-- visualize `gene expression trends`_ along specific lineages while accounting for the continuous nature of
-  fate determination
-- identify potential `driver genes`_ for each identified cellular trajectory
-
-Why is it called "CellRank"?
-----------------------------
-CellRank **does not** rank cells, we gave the package this name because just like Google's original `PageRank`_
-algorithm, it works with Markov chains to aggregate relationships between individual objects (cells vs. websites)
-to learn about more global properties of the underlying dynamics (initial & terminal states and fate probabilities vs.
-website relevance).
+Citing CellRank
+---------------
+If you find CellRank useful for your research, please check out :doc:`citing CellRank <about/cite>`.
 
 Support
 -------
-We welcome your feedback! Feel free to open an `issue <https://github.com/theislab/cellrank/issues/new/choose>`_, send
-us an `email <mailto:info@cellrank.org>`_ or `tweet`_ if you encounter a bug, need our help or just want to make a
-comment/suggestion.
+We welcome your feedback! Feel free to open an `issue`_ or send us an `email <mailto:info@cellrank.org>`_ if you
+encounter a bug, need our help or just want to make a comment/suggestion.
+
+CellRank in publications
```

Comment:
Would merge it somewhere in the introduction, this page has too many small sections and I felt overloaded.

Reply:
ok.

Reply:
did that.